### PR TITLE
fix: add missing controllerConfigRef on migration

### LINKED ISSUE #36: Missing controllerConfigRef in family migrator
### What happened?
When executing the `family-migrator` binary, the manifests generated do not contain the parameter `controllerConfigRef:` necessary to work the provider authentication. 

### How can we reproduce it?
 Generate the binary using the `make` command
 Execute `./family-migrator generate managed --plan-path output/out.yaml`

### PR DESCRIPTION
**Pull request by haooliveira84**
### Description of your changes

Fixes #36 

I have:

- [x] Run `make reviewable test` to ensure this PR is ready for review.

### CODE REVIEW

File: pkg/converter/configuration/converter.go [hunk 1]
```diff
@@ -264,6 +265,10 @@ func (pf *ProviderPkgFamilyParameters) ProviderPackageV1(p xppkgv1.Provider) ([]
 			continue
 		}
 		if extractProviderNameFromPackageName(p.Spec.PackageSpec.Package) == pf.Monolith {
+			cc := p.Spec.ControllerConfigReference
+			controllerConfigReference := xppkgv1.ControllerConfigReference{
```
Comment (sergenyalcin) [inline pkg/converter/configuration/converter.go:269]: We may consider adding a nil check here.

**Review by sergenyalcin — COMMENTED** [review]
Thanks @haooliveira84 I left a comment.

File: pkg/converter/configuration/converter.go (hunk 1 above)
Comment (haooliveira84) [reply pkg/converter/configuration/converter.go]: done @sergenyalcin

**Review by sergenyalcin — APPROVED** [review]
Thanks @haooliveira84 LGTM!

**Pull request merged**